x-pack/plugins/observability_solution/infra/public/components/ml/anomaly_detection/anomalies_table/anomalies_table.tsx

```diff
@@ -119,13 +135,44 @@ const AnomalyActionMenu = ({
     closeFlyout();
   }, [jobId, onViewChange, startTime, type, influencers, influencerField, closeFlyout]);
 
+  const showInHosts = useCallback(() => {
+    const newFilter = buildCombinedAssetFilter({
+      field: HOST_NAME_FIELD,
+      values: influencers,
+      dataView: metricsView?.dataViewReference,
+    });
+
+    const addHostFilter = (filter: Filter[] | Filter) => {
+      filterManagerService.removeAll();
+      filterManagerService.addFilters(filter);
+    };
+
+    if (!hostName || !closeFlyout) {
+      hostLocator?.navigate({
```

Comment:
nit: We can call it `hostsLocator` to keep it consistent

Reply:
Thx. Updated.